README.md

```diff
@@ -28,7 +28,7 @@ docker-compose up -d
 docker-compose exec stackstorm bash
 ```
 
-Open `https://localhost` in your browser. StackStorm Username/Password can be found in: `cat conf/stackstorm.env`
+Open `https://localhost` in your browser. StackStorm Username/Password can be found in: `cat ~/.st2/config`
```

Comment:
`conf/stackstorm.env` is a correct file to search for st2 username/password, see https://github.com/StackStorm/st2-docker/search?q=stackstorm.env&unscoped_q=stackstorm.env

This file looks like being populated based on `make env`.

Reply:
Hi @armab,

sorry for the confusion. I thought it's about the location in the container. To be honest, it's way more useful the way it is - hinting to the location on the host and not in the container. 

So I'll close the PR and thanks again!